tests/Dfe.PlanTech.Web.E2ETests/cypress.config.js

```diff
@@ -2,6 +2,7 @@ const { defineConfig } = require('cypress');
 const { loadAndSave, readFromJson } = require("./cypress/contentful-data-loader");
 
 module.exports = defineConfig({
+    numTestsKeptInMemory: 5,
```

Comment:
Whats this for?

Reply:
When the tests run over about 6 minutes they crash chrome because of memory consumption. It's been a known cypress issue forever. I'd like to take this back out and I think the refactoring ticket I'm looking at now (219929) will fix it but at the moment it seems to be the only way to get the tests to run all the way through without crashing.